setup.py

```diff
@@ -55,7 +53,8 @@
         "python-magic>=0.4.24,<0.5;platform_system!='Windows'",
         "python-magic-bin>=0.4.14,<0.5;platform_system=='Windows'",
     ],
-    "arrow": ["pyarrow>=7.0,<9.0"],
+    "arrow": ["pyarrow>=9.0,<10.0"],
```

Comment:
quietly updating py-arrow ?

Reply:
You did update that in the frontend so I am going to update it in the backend :D